code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -155,6 +155,7 @@ var/global/list/robot_modules = list(
 	src.modules += new /obj/item/device/flash(src)
 	src.modules += new /obj/item/weapon/crowbar/cyborg(src)
 	src.modules += new /obj/item/weapon/extinguisher(src)
+	vr_new() // For modules in robot_modules_vr.dm

```

Comment:
Add "Vorestation Edit" here so we can easily find it if we ever need to update this.